src/blocks.h

```diff
@@ -77,13 +76,13 @@ namespace libsemigroups {
     //! occurs earlier in \p blocks. None of this is checked.  The parameter \p
     //! blocks is not copied, and is deleted by the destructor Blocks::~Blocks.
     //!
-    //! The parameter \p lookup must have length equal to the maximum value in
-    //! \p blocks, this maximum is also the number of blocks in the partition. A
-    //! value \c true in position \f$i\f$ indicates that the \f$i\f$th block is
-    //! signed and \c false that it is unsigned.
+    //! The parameter \p lookup must have length equal to the number of
```

Comment:
Same again, please retain "the maximum value in \p blocks" in addition to the different wording you used.